src/models/media.ts

```diff
@@ -0,0 +1,32 @@
+import type { SupportedMediaType } from '~/constants/media'
+
+export interface Tag {
+  name: string
+}
+
+export interface ApiMedia {
+  id: string
+  title: string
+
+  creator: string
+  creator_url: string
```

Comment:
I think `creator_url` is optional. Not all creators have urls in the catalog.

Reply:
Made optional in 5927df4.